lib/notes.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
```

Comment:
file should be called Note.js. Capitalized and singular.

Reply:
Fixed to singular.